### ISSUE TITLE
default to LVM in disk recipe

### ISSUE THREAD

**Issue by superm1** [dell/dell-recovery#29]
If possible, it would be ideal to switch from ext4 to LVM.  
Switching to LVM will allow later expansion and support of multiple disks to be easier.

**Comment by kevinkellyspacey** [comment]
I'd like to try this kind of change. BTW, do we get some request to support this modification?

**Comment by superm1** [comment]
Yes there have been some inquiries about why we're not using LVM from the services team.

I don't know if this is supportable on a 16.04 base, if it's not you may need to use a 17.04 base (and some code that will test that it's 17.04 or later).

**Comment by superm1** [comment]
To save yourself trouble, I do think it's better to start with a 17.04 base and we can target this as an 18.04 feature.  I've started up a wiki page that you can track progress on the various milestones:
https://github.com/dell/dell-recovery/wiki/Multiple-disk-support
